src/hotspot/share/opto/vectorIntrinsics.cpp

```diff
@@ -331,6 +331,15 @@ bool LibraryCallKit::inline_vector_nary_operation(int n) {
 //  Sh ShuffleIota(Class<?> E, Class<?> ShuffleClass, Vector.Species<E> s, int length,
 //                  int start, int step, int wrap, ShuffleIotaOperation<Sh, E> defaultImpl)
 bool LibraryCallKit::inline_vector_shuffle_iota() {
+  // TODO shuffle is not supported on SVE
```

Comment:
This check is no necessary? Since SVE does not have Op_VectorLoadConst yet, arch_supports_vector() check below should return false.